configuration/pih/pih-config-mirebalais-humci.json

```diff
@@ -16,6 +16,7 @@
     "dataExports",
     "dbevent",
     "dispensing",
+    "drugRehab",
```

Comment:
I think adding this will crash HUM-CI if it's merged in before the drug rehab component is added to PIH Core?

Reply:
@mogoodrich Good point.  It didn't crash on my local, but I've removed drugRehab.  Might add it back with the other code next week if we need a new form.